experiments.py

```diff
@@ -111,6 +111,40 @@ def run_accuracy_experiment(path: Path = Path('./dataset/')):
     preds = logits_image.argmax(axis=1)
 
     
+    # Calculating and printing accuracy
+    acc = (preds == labels).mean()
+    print(f"CLIP accuracy on Oxford-IIIT Pets: {acc * 100:.3f}%")
+
+
+    # Creating prompts
+    class_names = [
+        "Abyssinian", "American Bulldog", "American Pit Bull Terrier", "Basset Hound",
+        "Beagle", "Bengal", "Birman", "Bombay", "Boxer", "British Shorthair", "Chihuahua",
+        "Egyptian Mau", "English Cocker Spaniel", "English Setter", "German Shorthaired",
+        "Great Pyrenees", "Havanese", "Japanese Chin", "Keeshond", "Leonberger", "Maine Coon",
+        "Miniature Pinscher", "Newfoundland", "Persian", "Pomeranian", "Pug", "Ragdoll", 
+        "Russian Blue", "Saint Bernard", "Samoyed","Scottish Terrier", "Shiba Inu", "Siamese", 
+        "Sphynx", "Staffordshire Bull Terrier", "Wheaten Terrier", "Yorkshire Terrier"
+    ]
+    prompts = [ f"A photo of a {c}" for c in class_names ]
```

Comment:
mano, não era pra ter nada disso aqui não

o prompt que o fessô pediu pro experimento de acurácia é só "A photo of a {c}, a type of pet"

Reply:
daria pra tirar toda essa segunda parte do `run_accuracy_experiment`